src/odoc/bin/main.ml

```diff
@@ -601,81 +601,49 @@ end = struct
           match f x with Some _ as result -> result | None -> find_map ~f l)
     in
     match l with
-    | [] -> Ok None
-    | _ -> (
-        match
-          find_map
-            ~f:(fun (pkg, path) ->
-              if Fpath.is_prefix path o then Some pkg else None)
-            l
-        with
-        | Some _ as r -> Ok r
-        | None -> Error `Not_found)
+    | [] -> None
+    | _ ->
+        find_map
+          ~f:(fun (pkg, path) ->
```

Comment:
These names are slightly misleading as `pkg` might actually be a lib. I know that was also the case before, but we might as well fix that now.